gidgethub/abc.py

```diff
@@ -31,11 +31,13 @@ def __init__(
         *,
         oauth_token: Opt[str] = None,
         cache: Opt[CACHE_TYPE] = None,
+        base_url: str = sansio.DOMAIN,
```

Comment:
I've made this default to `sansio.DOMAIN`, and updated the relevant documentation.